src/kremlib_base.h

```diff
@@ -23,7 +23,7 @@
 #ifndef __KREMLIB_BASE_H
 #define __KREMLIB_BASE_H
 
-#include <inttypes.h>
+#include <stdint.h>
```

Comment:
Due to the fact that this file is generated, we should have something like an automatic tool (`patch`?) to apply this kind of update and use `dune` with the promotion to apply as soon as possible this update. We can post-pone this requirement but we should make an issue about that.